### DISCUSSION TITLE
Packages/com.unity.ugui/manual/StyledText

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#81] [Notes]
# Packages/com.unity.ugui/manual/StyledText

https://docs.unity3d.com/Packages/com.unity.ugui@latest/index.html?subfolder=%2Fmanual%2FStyledText.html

**Comment by vertxxyz** [comment]
To escape a rich text tag replace the angle bracket characters with `U+3008` and `U+3009`.
You can also escape a rich text tag by placing a redundant one immediately after the opening bracket like:
`<<b></b>color=red>red<<b></b>/color>`
Which will print:
`<color=red>red</color>`
This can be helpful when you wish to display these tags in the console.

**Comment by mvi** [comment]
Note you can use the [`ColorUtility`](https://docs.unity3d.com/ScriptReference/ColorUtility.html) to convert a Unity color to the hex sequence needed for the color tag.  
For example:  

```csharp
Color myColor = new Color(1, 0.77f, 1);  
string foo = $"<color=#{ColorUtility.ToHtmlStringRGBA(myColor)}>Example</color>";
```
